app/views/admin/pages/new.html.erb

```diff
@@ -1,41 +1,41 @@
 <% @page_title = 'Add new page' %>
 
-<%= form_for :page do |f| %>
+<%= form_for :page, url: admin_page_new_path, method: :post do |f| %>
   <p>
-    Name:
+    <%= f.label :name %>
     <br><%= f.text_field :name %>
   </p>
   <p>
-    Description:
+    <%= f.label :description %>
     <br><%= f.text_field :description %>
   </p>
   <p>
-    Title:
+    <%= f.label :title %>
     <br><%= f.text_field :title %>
   </p>
   <p>
-    Slug:
+    <%= f.label :slug %>
     <br><%= f.text_field :slug %>
   </p>
   <p>
-    Template:
+    <%= f.label :template %>
```

Comment:
I'm not sure if the `<label>` `for` attributes generated on lines 21 and 26 will be correct. Clicking on the labels might not focus the associated `<select>`. It may need to be `<%= f.label :template_id, "Template" %>`.